interfaces/builtin/anbox_support.go

```diff
@@ -0,0 +1,512 @@
+
+    # starting the container
+    /run/lxc/ rw,  # should be anbox specific
+    /run/lxc/lock/ rw,
+    /run/lxc/lock/var/ rw,
+    /run/lxc/lock/var/snap/ rw,
+    /run/lxc/lock/var/snap/@{SNAP_NAME}/ rw,
+    /run/lxc/lock/var/snap/@{SNAP_NAME}/common/ rw,
+    /run/lxc/lock/var/snap/@{SNAP_NAME}/common/containers/{,**/} rw,
+    /run/lxc/lock/var/snap/@{SNAP_NAME}/common/containers/.default rwk,
+
+    mount options=(rw, bind) /var/snap/@{SNAP_NAME}/common/{,**/} -> /var/snap/@{SNAP_NAME}/common/lxc/{,**/},
+    mount options=(rw, bind) /var/snap/@{SNAP_NAME}/common/lxc/proc/sysrq-trigger -> /var/snap/@{SNAP_NAME}/common/lxc/proc/sysrq-trigger,
+    mount options=(ro, nosuid, nodev, noexec, remount, bind) -> /var/snap/@{SNAP_NAME}/common/lxc/proc/sysrq-trigger,
+    mount options=(rw, rbind) /var/snap/@{SNAP_NAME}/common/{,**/} -> /var/snap/@{SNAP_NAME}/common/lxc/{,**/},
+    mount fstype=tmpfs -> /var/snap/@{SNAP_NAME}/common/lxc/**/,
+    mount fstype=proc -> /var/snap/@{SNAP_NAME}/common/lxc/proc/,
+    mount options=(ro, nosuid, nodev, noexec, remount, bind) -> /var/snap/@{SNAP_NAME}/common/lxc/proc/sys/{,**/},
+    mount options=(rw, move) /var/snap/@{SNAP_NAME}/common/lxc/**/ -> /var/snap/@{SNAP_NAME}/common/lxc/**/,
+    mount options=(rw, nosuid, nodev, noexec) -> /var/snap/@{SNAP_NAME}/common/lxc/sys/,
+    mount options=(ro, nosuid, nodev, noexec, remount, bind) -> /var/snap/@{SNAP_NAME}/common/lxc/sys/,
+    mount options=(rw, nosuid, nodev, noexec, remount, bind) -> /var/snap/@{SNAP_NAME}/common/lxc/sys/{,**/},
+    mount fstype=sysfs -> /var/snap/@{SNAP_NAME}/common/lxc/**/,
+
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/full -> /var/snap/@{SNAP_NAME}/common/lxc/dev/full,
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/null -> /var/snap/@{SNAP_NAME}/common/lxc/dev/null,
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/random -> /var/snap/@{SNAP_NAME}/common/lxc/dev/random,
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/urandom -> /var/snap/@{SNAP_NAME}/common/lxc/dev/urandom,
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/tty -> /var/snap/@{SNAP_NAME}/common/lxc/dev/tty,
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/zero -> /var/snap/@{SNAP_NAME}/common/lxc/dev/zero,
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/pts/* -> /var/snap/@{SNAP_NAME}/common/lxc/dev/pts/*,
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/pts/0 -> /var/snap/@{SNAP_NAME}/common/lxc/dev/console,
+
+    pivot_root /var/snap/@{SNAP_NAME}/common/lxc/ -> /var/snap/@{SNAP_NAME}/common/lxc/,
+
+    # post pivot_root
+    # @{HOME} doesn't work here
+    mount options=(rw, bind) /home/*/snap/@{SNAP_NAME}/** -> /var/snap/@{SNAP_NAME}/common/lxc/dev/**,
+    mount options=(rw, bind) /home/*/snap/@{SNAP_NAME}/** -> /var/snap/@{SNAP_NAME}/common/devices/*,
```

Comment:
Why are you mounting devices from the user's home? Why are there devices in the user's home?